db/migrate/20151222011145_devise_create_users.rb

```diff
@@ -1,42 +0,0 @@
-class DeviseCreateUsers < ActiveRecord::Migration
```

Comment:
Was this old cruft?

Reply:
yeah, we made the devise migration before we fully swapped over to mongoid, so this (and possibly a few of the other generated files) aren't ever going to be used by anything again.